src/app/api/services/dot-confirmation/dot-confirmation.service.ts

```diff
@@ -13,6 +13,7 @@ import { ConfirmationService } from 'primeng/primeng';
 
 @Injectable()
```

Comment:
I want to leave a testimonial here:

"I don't like this solution, we should implement the ```p-dialog``` and ```p-confirmDialog```" Because they are different purpose for each.

I guess we can do that later.